DQM/TrackingMonitor/src/TrackSplittingMonitor.cc

```diff
@@ -148,15 +156,6 @@ void TrackSplittingMonitor::bookHistograms(DQMStore::IBooker & ibooker,
 void TrackSplittingMonitor::beginJob(void) 
```

Comment:
Hi Slava,
can you confirm that the "override" is to be added only in the analyze 
method in the header?

FF

Il 14/10/2015 17.15, Slava Krutelyov ha scritto:
>
> In DQM/TrackingMonitor/src/TrackSplittingMonitor.cc 
> <https://github.com/cms-sw/cmssw/pull/11792#discussion_r42007051>:
>
> > @@ -148,15 +156,6 @@ void TrackSplittingMonitor::bookHistograms(DQMStore::IBooker & ibooker,
> >  void TrackSplittingMonitor::beginJob(void)
>
> I suggest to remove beginJob and ednJob methods completely: they are 
> bogus for this module type (framework will never call them, either 
> single thread or multithread). Also, add |override| in the header file 
> to the virtual derived methods.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/11792/files#r42007051>.
>



---
Questa e-mail è stata controllata per individuare virus con Avast antivirus.
https://www.avast.com/antivirus